source/docs/beta/beta-getting-started/accessing-the-beta-project.rst

```diff
@@ -0,0 +1,36 @@
+Accessing the Beta Project
+==========================
+
+.. note::
+   Only accepted Beta Test teams will have access to the Beta project.
```

Comment:
```suggestion
```

Reply:
This is not specific in they styleguide and does not match the example here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html

Reply:
It's more for what matches elsewhere in the docs.